### ISSUE TITLE
 External Storage：webdav can auto support Thumbnail ??

### ISSUE THREAD

**Issue by xujiehehe** [owncloud/ocis#7287]
I use Extended Storage WebDAV to upload an image to a disk, but I cannot automatically display this image in my browser and can only download it. OwnCloud WebDAV does not have the ability to generate image previews, or is there a need for special settings? I have not found any documentation that describes this

**Comment by dragonchaser** [comment]
This is a bit vague, can you add more details? Which version of ocis are you running? Can you add some details about your configuration?

**Comment by xujiehehe** [comment]
@dragonchaser Thanks.

I installed OwnCloud using the Docker method, and the version used is owncloud/server: 10.12. I have inserted an external hard drive on my machine and deployed the webdav service using the webdav Docker. The version used is ugeek/webdav: arm64. On the settings page of owncloud, it is set to enable external storage and preview is enabled. I uploaded image files to this directory through the UI, but I was unable to generate a preview image.

system state：
Installed 1
Maintenance
NeedsDbUpgrade
Version 10.12.1.3
Versionstring 10.12.1
Editing Community
Productname ownCloud
Product ownCloud
Hostname 056e7cda47c8

Docker Running：
CONTAINER ID  IMAGE                           COMMAND            CREATED        STATUS                    PORTS                              NAMES
056e7cda47c8  owncloud/server:10.12   "/usr/bin/entrypoint…"   14 hours ago   Up 54 minutes    0.0.0.0:6081->8080/tcp, :::6081->8080/tcp    owncloud
d0953e3ab767  ugeek/webdav:arm64      "/bin/sh -c '/entryp…"   2 weeks ago    Up 54 minutes    0.0.0.0:20000->80/tcp, :::20000->80/tcp     e0cc60bd-c10c-4dab-b2be-5781f6cd4140

Settings page ：
![1695178792857](https://github.com/owncloud/ocis/assets/5696783/936193c1-fe27-437e-a396-4c79f199a8a9)

Upload File Page：(**This page cannot preview images. Does owncloud not support generating image thumbnails in this way? I have also installed nextcloud, and this feature is fine**)
![1695178852087](https://github.com/owncloud/ocis/assets/5696783/11f11d50-7646-4446-b41a-c867e7225670)
